src/main/java/hudson/plugins/s3/Destination.java

```diff
@@ -1,9 +1,9 @@
 package hudson.plugins.s3;
 
-import hudson.model.Run;
-
 import java.io.Serializable;
 
+import hudson.model.Run;
```

Comment:
Try not to re-order imports on files you are not changing. If you use IJ, there is a settings for that. I'd be surprise if Eclipse didn't have one similar.